app/src/main/java/com/github/se/cyrcle/model/report/ReportedObject.kt

```diff
@@ -0,0 +1,23 @@
+package com.github.se.cyrcle.model.report
+
+import com.github.se.cyrcle.model.parking.ParkingReportReason
+import com.github.se.cyrcle.model.review.ReviewReportReason
+
+sealed class ReportReason {
+  data class Parking(val reason: ParkingReportReason) : ReportReason()
```

Comment:
ps

Reply:
Better thanks, this confusion also happenend because I didn't understand this was only for flagged item.